pkgs/top-level/python-packages.nix

```diff
@@ -25136,16 +25151,17 @@ in modules // {
   };
 
   searx = buildPythonPackage rec {
-    name = "searx-0.8.1";
+    name = "searx-${version}";
+    version = "0.9.0";
 
     src = pkgs.fetchurl {
-      url = "https://github.com/asciimoo/searx/archive/v0.8.1.tar.gz";
-      sha256 = "0z0s9n8iblrw7y5xrh2apzsazkgm4vzmxn0ckw4yfiya9am8zk32";
+      url = "https://github.com/asciimoo/searx/archive/v${version}.tar.gz";
+      sha256 = "0hbwvyah7mml452fyr7j14dqsbp1xsk7dglqpvsgg5cq0aiq9yp5";
     };
```

Comment:
```
building path(s) ‘/nix/store/0kdv4m6k8jbdp2wnvd8pp4xmirri9c8m-python2.7-searx-0.9.0’
unpacking sources
unpacking source archive /nix/store/ghdzg3c785zx2ik6jd5ad4f43m7fw4kp-searx-0.9.0.tar.gz
source root is searx-0.9.0
setting SOURCE_DATE_EPOCH to timestamp 1464090064 of file searx-0.9.0/setup.cfg
patching sources
configuring
building
Traceback (most recent call last):
  File "nix_run_setup.py", line 6, in <module>
    exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\\r\\n', '\\n'), __file__, 'exec'))
  File "setup.py", line 20, in <module>
    requirements = map(str.strip, open('requirements.txt').readlines())
IOError: [Errno 2] No such file or directory: 'requirements.txt'
builder for ‘/nix/store/gnhcvdvgqf4a8brnizh3rfl7amzgpygi-python2.7-searx-0.9.0.drv’ failed with exit code 1
```

Now I'm confused. The setup.py is opening requirements.txt but that's not in the tarball at pypi. :frowning: